crates/db_views/src/comment_view.rs

```diff
@@ -310,8 +309,7 @@ pub struct CommentQuery<'a> {
   pub local_user: Option<&'a LocalUserView>,
   pub search_term: Option<String>,
   pub saved_only: Option<bool>,
-  pub is_profile_view: Option<bool>,
-  pub show_deleted_and_removed: Option<bool>,
+  pub is_profile_view: bool,
```

Comment:
If you're going to change this one, might as well change all the other `Option<bool>` to `bool` . Same goes for `post_view.rs`

Reply:
Then the api code gets messy because it has to convert Option<bool> to bool. So it only makes sense to change both together, can do that in another PR.

Reply:
Okay, I'll make an issue for that.